styles/Series.module.scss

```diff
@@ -0,0 +1,68 @@
+@import './colors';
+
+:export {
+  bodyColor: #{$primary-900};
+}
+
+.root {
+  min-height: 100vh;
+  background-color: $grey-100;
+}
+
+.logoBar {
+  width: 100vw;
+  padding: 1rem 2rem;
+
+  background-color: $primary-900;
+  color: $white;
+}
+
+.prxLogo {
+  width: 3.6rem;
+}
+
+.header {
+  width: 100vw;
+  height: 300px;
+
+  background-color: $grey-500;
+}
+
+.content {
+  width: 100vw;
+  max-width: 900px;
+  padding: 2rem 0rem;
+  margin: auto;
+}
+
+.episodeContainer {
+  width: 56rem;
```

Comment:
Change to use `max-width` to be responsive on screens smaller than the designed width.

Reply:
We can now get rid of this style.